File: Flutter App/unicoVehicle/lib/Models/Order.dart
import './Vehicle.dart';
import './Showroom.dart';
import './Status.dart';
import './Customer.dart';

class Order {
  final int orderId;
  final Vehicle vehicle;
  final Showroom showroom;
  final Status status;
  final Customer customer;
  final double finalPrice;
  final bool hasLoan;
  final DateTime deliveryDate;

  Order({
    this.customer,
    this.deliveryDate,
    this.finalPrice,
    this.hasLoan,
    this.orderId,
    this.showroom,
    this.status,
    this.vehicle,
  });
}

File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/Order.cs
using System;
namespace UnicoVehicle.DTO
{
    public class Order
    {
        public int OrderId { get; set; }
        public Vehicle Vehicle { get; set; }
        public Showroom Showroom { get; set; }
        public Status OrderStatus { get; set; }
        public Customer Customer { get; set; }
        public double FinalPrice { get; set; }
        public bool hasLoan { get; set; }
        public DateTime DeliveryDate { get; set; }
    }
}
